src/components/form/form.scss

```diff
@@ -0,0 +1,6 @@
+@import '../../style/variables';
+
+limel-select {
+    display: block;
+    margin-bottom: pxToRem(38);
```

Comment:
![image](https://user-images.githubusercontent.com/154725/73058983-026d7b00-3e95-11ea-954f-e02d2d4976ff.png)

I think we need to only apply this margin if there's a helper-text. It creates a huge gap otherwise:

![image](https://user-images.githubusercontent.com/154725/73059052-28931b00-3e95-11ea-80d7-3009c99a3610.png)

Reply:
And it probably should go in the limel-select css, unless there's a particular reason it's only needed when used in a form?